docs/book/installation/sylius_plus_upgrading.rst

```diff
@@ -0,0 +1,39 @@
+.. rst-class:: plus-doc
+
+Upgrading Sylius Plus
+=====================
+
+Sylius regularly releases new versions, usually every two weeks.
+Each release comes with an ``UPGRADE.md`` file, which is meant to help in the upgrading process.
+
+1. **Update the Sylius Plus version constraint by modifying the ``composer.json`` file:**
```

Comment:
Can we expose more clearly, that `0.21` is just an example? 
```suggestion
For example, if one would like to update to version `0.21`:

1. **Update the Sylius Plus version constraint by modifying the ``composer.json`` file:**
```

Reply:
it's same as in the OS upgrade guide:
https://docs.sylius.com/en/latest/book/installation/upgrading.html